vpd-tool/src/vpd_tool.cpp

```diff
@@ -85,4 +86,47 @@ int VpdTool::readKeyword(const std::string& i_vpdPath,
     }
     return l_rc;
 }
+
+int VpdTool::writeKeyword(const std::string& i_vpdPath,
+                          const std::string& i_recordName,
+                          const std::string& i_keywordName,
+                          const std::string& i_keywordValue,
+                          const bool i_onHardware,
+                          const std::string& i_filePath)
+{
+    int l_rc = constants::FAILURE;
+    try
+    {
+        if (i_vpdPath.empty() || i_recordName.empty() || i_keywordName.empty())
+        {
+            throw std::runtime_error("Received input is empty.");
+        }
+
+        if (!i_filePath.empty())
+        {
+            // ToDo: Take keyword value from the file
```

Comment:
For "to dos", are we using "TODO" or "ToDo" ? We should be consistent and stick with one, so that in future we can `grep` the repo and get all the "to dos".

Reply:
From the doxygen I am not able to figure out what is happening here. Why it reads from file? What if the kwd value is passed, what happens in that case?

Reply:
user can give  keyword value either using -V option or using --file option

Ex:
 On DBus: vpd-tool -w -O <DBus Object Path> -R <Record Name> -K <Keyword Name> -V <Keyword Value>
  On DBus, take keyword value from file:
       vpd-tool -w -O <DBus Object Path> -R <Record Name> -K <Keyword Name> --file <File Path>

Reply:
Which part of doxygen says that? How am I supposed to know that?
Why don't you overload the API as per both the cases?